src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -164,7 +165,7 @@ public function toolbarAction(Request $request, $token)
 
         $url = null;
         try {
-            $url = $this->generator->generate('_profiler', ['token' => $token]);
+            $url = $this->generator->generate('_profiler', ['token' => $token], RouterInterface::ABSOLUTE_URL);
```

Comment:
Please use `UrlGeneratorInterface` here, which is the actual source of the constant.

Reply:
Alright